aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2562,6 +2562,17 @@ Tensor slice(
   return result;
 }
 
+Tensor slice_inverse_symint(
+    const Tensor& self,
+    const Tensor& base,
+    int64_t dim,
+    c10::optional<SymInt> start,
```

Comment:
These should be const ref especially since they aren't used. You may want to mark them as unused by commenting out the variable names. 
```suggestion
    c10::optional<SymInt> /*start*/,
```

Reply:
Thanks for the review! I don't think I change to const ref here because the signature has to match the one in the generated header, but I can comment out the unused names.